components/debug/memory-viewer.tsx

```
@@ -0,0 +1,413 @@
+    time: string
+    systemMemoryPercent: number
+    chromeMemory: number
+    ffmpegMemory: number
+    chromeCpu: number
+    chromeProcessCount: number
+}
+
+type SubTabType = "memory" | "performance"
+
+// Custom tooltip component with larger fonts and size
+const CustomTooltip = ({ active, payload, label, config }: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)